coalib/tests/results/ResultTest.py

```diff
@@ -28,15 +28,22 @@ def test_origin(self):
     def test_string_conversion(self):
         uut = Result('a', 'b', 'c')
         self.assertEqual(str(uut),
-                         "Result:\n origin: 'a'\n file: 'c'\n line nr: None\n"
-                         " severity: 1\n diffs: None\n'b'")
+                         "Result:\n"
+                         " id: {}\n"
+                         " origin: 'a'\n"
+                         " file: 'c'\n"
+                         " line nr: None\n"
+                         " severity: 1\n"
+                         " diffs: None\n"
+                         "'b'".format(uut.id))
         self.assertRegex(
             repr(uut),
-            "<Result object\\(origin='a', file='c', line_nr=None, "
+            "<Result object\\(id=-?[0-9]+, origin='a', file='c', line_nr=None, "
                 "severity=NORMAL, message='b'\\) at 0x[0-9a-fA-F]+>")
-        self.assertEqual(
+        self.assertRegex(
             Result("origin", "message", "file", line_nr=1).__str__(),
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unexpected keyword argument 'line_nr' in constructor call](https://www.codacy.com/app/lasse/coala/file/1100294817/issues/source?bid=2303100&fileBranchId=2892719#l44)